ideas/170-wallet-improvements.md

```diff
@@ -0,0 +1,74 @@
+## Preamble
+
+    Idea: 170
+    Title: Wallet Improvements
+    Status: Draft
+    Created: 2018-04-13
+    
+## Summary
+
+Status Wallet does not fully support ERC20 tokens (requesting and transaction history not available)
+and generally the transactions UX is different between Chat and Wallet. The intention of this swarm
+is to wrap up those inconsistencies.
+
+## Swarm Participants
+- Lead Contributor: [@goranjovic](https://github.com/goranjovic)
+- Contributor: 
+- Testing & Evaluation: [@asemiankevich] (https://github.com/asemiankevich)
+- PM: [@chadyj](https://github.com/chadyj)
+- UX: 
```

Comment:
@hesterbruikman and @jpbowen can bring huge insight here with UXR!

@denis-sharypin @andmironov are you interested?

Reply:
@chadyj yep! we together with @hesterbruikman  wanted to create such swarm. We are planning to have a call on Monday. So interested indeed!